build/luajit.zig

```diff
@@ -210,6 +243,11 @@ pub fn configure(b: *Build, target: Build.ResolvedTarget, optimize: std.builtin.
     return lib;
 }
 
+fn getPath(lazy_path: Build.LazyPath, src_builder: *Build, asking_step: ?*Step) []const u8 {
+    const p = lazy_path.getPath3(src_builder, asking_step);
```

Comment:
are users of the Zig build system supposed to call this function? I'm pretty sure they are not, right?

Reply:
Yes you are right, the function was added originally and then not used, I forgot to remove it.